xbmc/pvr/windows/GUIWindowPVRChannels.h

```diff
@@ -29,14 +29,14 @@ namespace PVR
     CGUIWindowPVRChannels(bool bRadio);
     virtual ~CGUIWindowPVRChannels(void) {};
 
-    bool OnMessage(CGUIMessage& message);
-    void GetContextButtons(int itemNumber, CContextButtons &buttons);
-    bool OnContextButton(int itemNumber, CONTEXT_BUTTON button);
-    bool Update(const std::string &strDirectory, bool updateFilterPath = true);
-    void UpdateButtons(void);
-    void ResetObservers(void);
+    virtual bool OnMessage(CGUIMessage& message) override;
```

Comment:
GUIWindowPVRChannels is not supposed to have any derived classes. A class with methods flagged as virtual implies that it can have subclasses.